### DISCUSSION TITLE
display result after each puzzle

### DISCUSSION THREAD

**Discussion by arnaule** [rodpolako/Chess-PGN-Trainer#8] [Ideas]
When loading a pgn with mutiple lines, it loads multiple puzzles (after using pgn-extract)
the result screen is displayed only after the end of the last puzzle.
It would be great to pause and display the result after each puzzle.
Ideally with a button to do the puzzle again.
Thank you!

**Comment by rodpolako** [comment]
I think I understand what you mean here.  If you are talking about showing your score (and not just the % complete) during the set that is something I can consider.  The ability to pause is already in place but perhaps I can add the ability to score your current score when paused.  

The design of the tool is to drill through a set as efficiently as possible so pausing after each puzzle to show your score or repeating puzzles would hinder that but I can see the potential of having the option to show your score/stats while paused.  I'll have to think on that one.  Thanks for the suggestion!
